src/lib/redux/slices/pitwallSlice/selectors.ts

```diff
@@ -30,6 +30,11 @@ export const getCurrentConditions = (state: ReduxState) =>
 
 export const getLiveTiming = (state: ReduxState) => state.pitwall.liveTiming;
 
+export const selectCurrentCar = (state: ReduxState) =>
+  state.pitwall.liveTiming.find(
+    (lt) => lt.carNumber === getSelectedTelemetryProvider(state)?.carNumber,
```

Comment:
does `carNumber` actually have a value? I remember adding it for doing things like this, but I don't recall actually wiring it up 🤔

Reply:
You are correct, this currently doesn't work. The attribute is `carNumber: null`. I changed now to use the name. Let me know, when the `carNumber` is populated, then I can change it back.